tests/virtualization/universal/hotplugging_network_interfaces.pm

```diff
@@ -38,7 +38,6 @@ sub add_virtual_network_interface {
         if (get_var('VIRT_AUTOTEST') && is_kvm_host) {
             $interface_model_option = '--model virtio';
         }
-        script_retry("ssh root\@$guest ip l | grep " . $virt_autotest::common::guests{$guest}->{macaddress}, delay => 60, retry => 10, timeout => 60);
```

Comment:
The hash  $virt_autotest::common::guests will be dropped.  The code change in the is preparatory step for the change in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15719.  

As change to many files,  now, I am thanking  combine the two. Drop this one. That's save my time, VR time, review time.  It might minimum misunderstanding.